io.github.khaled_0.TubeSync.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
missing a bunch of mandatory tags, validate it and add them https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation